src/arrayMethodJoin.js

```diff
@@ -4,8 +4,28 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] === null || this[i] === undefined) {
+        result += ',';
```

Comment:
```suggestion
        result += separator;
```
What if separator will be specified in arguments and it will be not a comma, but, for example, `/`?